### ISSUE TITLE
Sector management - add sector - not edit IATI source sectors

### ISSUE THREAD

**Issue by matmaxgeds** [somaliaims/Somali-AIMSUI#286]
To avoid making a messy situation, we need to treat the sectors (both the types and the sectors they contain) that are imported from IATI as not management user modifiable - otherwise if a management user deletes a sector, it will be re-added causing confusion etc, re-namings will be controversial etc.

So please set it so that if a sector type is imported from IATI, then it should not appear in the menu for 'create sector', nor should the sector types or sectors have a delete button available. The only option should be the re-naming of the sector type - but not re-naming of the sectors.

Basically, the only options for IATI imported sectors are to re-name the sector type, and to use them in the mappings.

**Comment by matmaxgeds** [comment]
closing as now (for the fix) duplicated here: https://github.com/somaliaims/Somali-AIMSUI/issues/433
